<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BROWSEME</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/ico.png</normaloff>:/ico.png</iconset>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabsBrowser">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabPages">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/ico.png</normaloff>:/ico.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Вкладки</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="btBack">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/ico5.png</normaloff>:/ico5.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btForward">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/ico6.png</normaloff>:/ico6.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="editPageUrl"/>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btReloadStop">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/ico4.png</normaloff>:/ico4.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="btAddPage">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/ico7.png</normaloff>:/ico7.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="QProgressBar" name="barLoadProgress">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>5</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>5</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QProgressBar::chunk {
	background-color: rgb(56,62,64);
}</string>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="5">
         <widget class="QTabWidget" name="tabsWebPages">
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabBrowserSettings">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/ico2.png</normaloff>:/ico2.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Настройки Браузера</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QGroupBox" name="gbBrowserHistory">
          <property name="title">
           <string>История браузера:</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QTreeWidget" name="treeBrowserHistory">
             <property name="columnCount">
              <number>2</number>
             </property>
             <attribute name="headerDefaultSectionSize">
              <number>170</number>
             </attribute>
             <attribute name="headerMinimumSectionSize">
              <number>170</number>
             </attribute>
             <attribute name="headerStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Заголовок/Сайт</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Дата</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbAllowings">
          <property name="title">
           <string>Разрешения:</string>
          </property>
          <widget class="QCheckBox" name="cbAllowJavaScripts">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>125</y>
             <width>291</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Разрешить исполнение сценариев JavaScript</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="cbAllowPlugins">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>35</y>
             <width>247</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string> Разрешить использование плагинов</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="cbAllowAutoImg">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>65</y>
             <width>328</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Разрешить автоматическую загрузку изображений</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="cbAllowJSClipboard">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>185</y>
             <width>291</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Разрешить JS доступ к буферу обмена</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="cbAllowJSOpen">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>155</y>
             <width>291</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Разрешить JS открывать окна</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="cbAllowInsecContent">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>215</y>
             <width>301</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Разрешить исполнять незащищенный контент</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="cbAllowGeoloc">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>245</y>
             <width>291</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Разрешить геолокацию</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="cbAllowSound">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>95</y>
             <width>291</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Разрешить воспроизведение звука</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="cbAllowDNT">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>275</y>
             <width>291</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Использовать заголовок DNT</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbProxy">
          <property name="title">
           <string>Прокси:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="cbProxyAuth">
             <property name="text">
              <string>Аутентификация по учетным данным</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbProxyAuth">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Учетные данные прокси:</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLineEdit" name="editProxyUser">
                <property name="placeholderText">
                 <string>Имя пользователя...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="editProxyPass">
                <property name="placeholderText">
                 <string>Пароль...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editProxyIp">
             <property name="placeholderText">
              <string>Адрес прокси...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbProxyPort">
             <property name="minimum">
              <number>10</number>
             </property>
             <property name="maximum">
              <number>65536</number>
             </property>
             <property name="value">
              <number>8080</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btTurnProxy">
             <property name="text">
              <string>Включить прокси</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/ico8.png</normaloff>:/ico8.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
